### ISSUE TITLE
Warning modal (hover) has bright yellow background making text unreadable

### ISSUE THREAD

**Issue by ThatOneCalculator** [rose-pine/zed#1]

**Comment by mikkurogue** [comment]
To fix this issue locally I added this to my settings.json:

```
"experimental.theme_overrides": {
    "warning.background": "#ea9d34",
    "warning.border": "#ea9d34",
    "warning": "#ea9d34", // this is just the underline on the warning
}
```

This seems to fix the issue of warnings being an unreadable mess, but I'm not sure if I used the correct pallet as this is the yellow from the Dawn pallet (Using Rosé Pine as my theme). This however seems to fix the issue for me, or at least make it readable:
![Screenshot 2024-06-29 at 23 46 01](https://github.com/rose-pine/zed/assets/5637842/d70856c2-eea4-46ea-b2f4-53a346b86512)

**Comment by SergoGansta777** [comment]
I think this is related. 
![CleanShot 2024-07-18 at 20 17 00@2x](https://github.com/user-attachments/assets/4fa74173-d96f-4fe3-88be-c8a746b4654a)

**Comment by ThatOneCalculator** [comment]
Fixed in v1.1.2
